src/App.css

```diff
@@ -0,0 +1,42 @@
+.App {
```

Comment:
Do we want this to be `css` or go ahead and use `scss`?

Reply:
scss